assets/stylesheets/transactions/transactions.css

```diff
@@ -0,0 +1,21 @@
+#transactions{
+	position: relative;
+	position: center;
+	background-color: #00000;
```

Comment:
not even a color :)

Reply:
hahaha thanks
I still have a long way to go